src/grt/src/GlobalRouter.cpp

```diff
@@ -3982,9 +3982,9 @@ void GlobalRouter::findBufferPinPostions(Net* net1,
                                          odb::Point& pin_pos2)
 {
   for (const Pin& pin1 : net1->getPins()) {
-    if (!pin1.isPort()) {
+    if (!pin1.isPort() && pin1.getITerm() != nullptr) {
       for (const Pin& pin2 : net2->getPins()) {
-        if (!pin2.isPort()) {
+        if (!pin2.isPort() && pin2.getITerm() != nullptr) {
```

Comment:
@maliberty In the current code, here's is the only part where the check is needed, since the other parts run before the incremental grt.